packages/forms/src/__tests__/FormikCheckboxField.spec.tsx

```diff
@@ -41,6 +42,40 @@ test('handles changes', async () => {
   expect(handleSubmit).toHaveBeenLastCalledWith({ agree: false });
 });
 
+test('format and parse value', async () => {
+  const handleSubmit = jest.fn();
+
+  const wrapper = renderFormField(
```

Comment:
make a better example with `active`/`inactive` enumerator values. you will find problems in type definitions.